lib/cli/run.js

```diff
@@ -241,6 +241,20 @@ exports.builder = yargs =>
       watch: {
         description: 'Watch files in the current working directory for changes',
         group: GROUPS.FILES
+      },
+      'watch-files': {
+        description: 'Comma separated list of paths or globs to watch',
```

Comment:
I'm unsure about "Comma separated list". Normally (see eg `--file`) we prefer to repeat the option.
I will do some testing ...

Reply:
I’m fine with either

Reply:
Please change to "List of paths or globs to watch"